### PR TITLE
Fix and renable safetydb importer

### PR DESCRIPTION
**Pull request by sbs2001**
Fixes https://github.com/nexB/vulnerablecode/issues/244

Signed-off-by: Shivam Sandbhor <shivam.sandbhor@gmail.com>

### CODE REVIEW

File: vulnerabilities/importer_yielder.py [hunk 1]
```diff
@@ -61,16 +61,16 @@
...
-#         'name': 'safetydb',
-#         'license': 'cc-by-nc-4.0',
-#         'last_run': None,
-#         'data_source': 'SafetyDbDataSource',
-#         'data_source_cfg': {
-#             'url': 'https://raw.githubusercontent.com/pyupio/safety-db/master/data/insecure_full.json',  # nopep8
-#             'etags': {}
-#         },
-#     },
+    {
```
Comment (pombredanne) [inline vulnerabilities/importer_yielder.py:64]: We need to have a way to selectively enable some non-free resources like this one. As long as this is under a cc-by-nc  license we cannot integrate these data by default IMHO 
@jayfk ping?

**Review by pombredanne — CHANGES_REQUESTED** [review]
This looks fine but we cannot have this enabled by default license-wise for now. How could we have a way to enable this on demand?

**Comment by sbs2001** [comment]
@pombredanne  from the discussion we had, you suggested we handle this  by using a config file which contains a list of licenses. 
If an importer's license is not present in this list either run/don't run the importer. 

I think this logic belongs in the `importer_yielder.py` file, no need to create another config file.

File: vulnerabilities/importer_yielder.py [hunk 2]
```diff
@@ -61,16 +61,16 @@
...
+        'name': 'safetydb',
+        'license': 'cc-by-nc-4.0',
+        'last_run': None,
+        'data_source': 'SafetyDbDataSource',
+        'data_source_cfg': {
+            'url': 'https://raw.githubusercontent.com/pyupio/safety-db/master/data/insecure_full.json',  # nopep8
+            'etags': {}
+        },
+    },
```
Comment (pombredanne) [inline vulnerabilities/importer_yielder.py:73]: Let's add instead a flag to disable this rather than commenting it out and leave this disabled.

**Comment by sbs2001** [comment]
Commits included in https://github.com/nexB/vulnerablecode/pull/259. Hence closing

**Pull request closed without merging**